deploy/040-controller.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       serviceAccount: kubevirt-csi-controller-sa
       priorityClassName: system-cluster-critical
       nodeSelector:
-        node-role.kubernetes.io/master: ""
+        node-role.kubernetes.io/worker: ""
```

Comment:
Actually, let's just remove this label entirely. THere's no reason why this controller needs to live on nodes with the master label now.

Reply:
ok, will do.